### ISSUE TITLE
cannot find API endpoint to create tenancy alteration

### ISSUE THREAD

**Issue by techleadcht** [reapit/foundations#11226]
**Which API call should we use to create a new tenancy alteration ("Add renewal, alteration or fee" in screenshot)?**

![image (2)](https://github.com/reapit/foundations/assets/153836949/89da983d-8715-42ce-bece-10907b0329ed)

It is clear that there is a POST endpoint to create a Renewal Negotiation but not an alteration.
![image](https://github.com/reapit/foundations/assets/153836949/1ef4c2dd-372c-4bae-b51f-08508e57e170)

Please advise.

**Comment by AshDeeming** [comment]
Hi @techleadcht 
For alterations on a tenancy please see the /tenancies/{id}/extensions 
![image](https://github.com/reapit/foundations/assets/61687775/910b2089-08f9-4963-be43-2f91cb11fd4a)

**Comment by AshDeeming** [comment]
Thanks for the details @techleadcht 
Could you advise the business case for this & i';; convert the ticket to a feature request for back end review

**Comment by techleadcht** [comment]
Yes, I am building a complex rent schedule using my own system and I want to be able to create Alterations on Reapit Cloud to reflect this. For example:
9 months tenancy
Initial frequency monthly for 2 months.
then
bi-monthly for 4 months
then
lump sum for the remaining 3 months.

**Comment by ritakapila** [comment]
Do we have a timescale on this one please
Thank you

**Comment by HollyJoyPhillips** [comment]
Internal Reference: PBI-5796
